admin-manual/maintenance/maintenance.rst

```diff
@@ -544,6 +544,58 @@ an alternative to using the dashboard.
 6. To update the list and continue, enter `u` when prompted to enter a value.
    Repeat the steps above until there are no more choices left to make.
 
+
+.. _transfer-wont-start:
+
+Transfer won't start
+^^^^^^^^^^^^^^^^^^^^
+
+Sometimes, a user may attempt to start a transfer and it will never seem to 
+initiate the Archivematica processes. There are a few issues to look out for 
+and investigate if this happens.
+
+1. File permissions
+
+  First, the issue may be related to file permissions in the transfer source 
+  directory. Check the permissions in the directory and on the files to ensure 
+  that all files can be read by Archivematica.
+
+2. System timeouts
+
+  If it is a large transfer, it may just be taking a long time to copy the files
+   and initially load them into the system, and the user can wait a bit longer 
+   and see if the processes begin after a bit of time.  It is also possible that 
+   it is taking a long time because some of the system timeouts are being 
+   exceeded and the transfer has failed. This can be verified by checking the 
+   Storage Service logs and by checking where the transfer exists on the 
+   filesystem.
+
+  Storage Service logs can be found in the 
+  `/var/log/archivematica/storage-service/` directory.
+
+  For inadequate timeouts, check the Storage Service configuration and adjust 
+  if necessary.
+
+3. Communication between Dashboard and Gearman
+
+  If the transfer has successfully moved to the shared Directory (i.e. it can be
+   found in sharedDirectory/watchedDirectories/activeTransfer/ folders), but it 
```

Comment:
`sharedDirectory/watchedDirectories/activeTransfer/` could be an inline literal or you could also use that other form that I can't remember how it's called, e.g. `:file:`sharedDirectory/dadada...` (interpreted text?).

Reply:
+1 for inline literal in this situation.